packages/server/lib/server.ts

```diff
@@ -76,6 +77,7 @@ const upload = multer({ storage: multer.memoryStorage() });
 // operator should run migrate.ts once before starting the service.
 if (NANGO_MIGRATE_AT_START === 'true') {
     await migrate();
+    await migrateRecords();
```

Comment:
missing in migrate.ts

Reply:
not sure what migrate.ts is for but yeah I will add it there

Reply:
https://github.com/NangoHQ/nango/pull/663